doc/rest.md

```diff
@@ -130,7 +130,7 @@ Body parameters:
       - JPEG (.jpg, .jpeg, with .jpg.aux.xml, .jgw, .jpgw, .jpegw or .wld)
       - any of above types in single ZIP file (.zip)
       - file names, i.e. array of strings
-   - it is allowed to publish time-series layer by setting time_regex parameter and sending one or more main raster files (compressed in one archive or uncompressed) with the same extension, color interpretation, bounding box, raster size, nodata value, mask flags, and data type name. Filename can be at most 210 characters long.
+   - it is allowed to publish time-series layer by setting time_regex parameter and sending one or more main raster files (compressed in one archive or uncompressed) with the same extension, color interpretation, bounding box, raster size, nodata value, mask flags, and data type name. Filename can be at most 210 characters long. Safe characters are `a-zA-Z0-9_.-`. Letters with diacritic are also possible, but diacritic is removed, and space ` ` is also possible, but converted to `_`. Other than latin scripts are not supported.
```

Comment:
Supported characters are latin letters (with or without diacritics), numbers, underscores, dashes, dots, and spaces. Other than latin scripts are not supported. Files are stored and published with slugified filename, where diacritic is removed from letters, and space ` ` is converted to underscore `_`.